tickers/src/main/kotlin/fund/cyber/markets/tickers/TickersApplication.kt

```diff
@@ -0,0 +1,55 @@
+package fund.cyber.markets.tickers
+
+import fund.cyber.markets.dto.TokensPair
+import fund.cyber.markets.kafka.JsonSerde
+import fund.cyber.markets.model.Trade
+import fund.cyber.markets.tickers.configuration.KafkaConfiguration
+import fund.cyber.markets.tickers.model.WindowKey
+import fund.cyber.markets.tickers.model.WindowStats
+import org.apache.kafka.common.serialization.Serdes
+import org.apache.kafka.streams.KafkaStreams
+import org.apache.kafka.streams.kstream.KStream
+import org.apache.kafka.streams.kstream.KStreamBuilder
+import org.apache.kafka.streams.kstream.TimeWindows
+import java.sql.Timestamp
+
+val configuration: KafkaConfiguration = KafkaConfiguration()
+
+fun main(args: Array<String>) {
```

Comment:
just answer question: If error occurred inside stream handlers, will application exit?

Reply:
yes, the application will exit if any exception occurs